packages/services/src/block_bundler.rs

```diff
@@ -402,30 +401,35 @@ mod tests {
         Ok(())
     }
 
+    fn merge_data(blocks: impl IntoIterator<Item = CompressedFuelBlock>) -> NonEmpty<u8> {
+        blocks
+            .into_iter()
+            .flat_map(|b| b.data)
```

Comment:
shouldn't we also include the height

Reply:
it's probably inside of the compressed block itself, should be checked @hal3e

Reply:
Compressed block contains `PartialBlockHeader` which has the height. https://docs.rs/fuel-core-types/latest/fuel_core_types/blockchain/header/struct.PartialBlockHeader.html